src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + getArea() + " sq. units, radius: "
+                + this.radius + " units, color: " + this.color);
```

Comment:
It's a good practice to use the getter method `getColor()` to access the `color` field instead of accessing it directly. This ensures better encapsulation and consistency with OOP principles.